include/UIKit/UIPopoverControllerDelegate.h

```diff
@@ -24,9 +24,11 @@
 #import <CoreGraphics/CGGeometry.h>
 
 @protocol UIPopoverControllerDelegate <NSObject>
+@optional
+
 - (void)popoverController:(UIPopoverController*)popoverController
     willRepositionPopoverToRect:(inout CGRect*)rect
-                         inView:(inout UIView* _Nonnull*)view;
```

Comment:
I think I'll have to make a few simple changes to the WYPopoverController headers to properly support these attributes throughout. I will change all of these in the next fixup commit.